src/sections/perguntas-frequentes/index.jsx

```diff
@@ -0,0 +1,250 @@
+            utilizadores poderem reportar qualquer outro sintoma que não está
+            actualmente listado.
+          </Typography>
+        </Collapse>
+        <Collapse title="Posso introduzir sintomas em nome de outra pessoa (ex.: um familiar)?">
+          <Typography variant="smallBody">
+            Na versão actual não é possível. Esperamos conseguir lançar em breve
+            novas funcionalidades que permitam introduzir os sintomas de um
+            familiar.
+          </Typography>
+        </Collapse>
+        <Collapse title="Gostava de partilhar os meus sintomas com os meus familiares - posso fazê-lo?">
+          <Typography variant="smallBody">
+            Não é possível partilhar os seus sintomas com outros utilizadores.
+          </Typography>
+        </Collapse>
+        <Collapse title="Tenho menos de 18 anos, posso usar a Covidografia?">
+          <Typography variant="smallBody">
+            Não. Para utilizar a Covidografia, é necessário dar consentimento
+            para o tratamento dos dados e, como tal, é necessário maioridade.
+            Para mais detalhes, consulte os nossos{" "}
+            <Button
+              href={`${data.site.siteMetadata.appSiteUrl}/#/terms-conditions`}
+              type="textLink"
+            >
+              termos de utilização
+            </Button>
+            .
+          </Typography>
+        </Collapse>
+        <Collapse title="Encontrei um erro na Covidografia - o que devo fazer?">
+          <Typography variant="smallBody">
+            Agradecemos todo e qualquer feedback sobre a Covidografia. Pedimos
+            que entre em contacto connosco através do endereço{" "}
+            <Button href="mailto:info@covidografia.pt" type="textLink">
+              info@covidografia.pt
+            </Button>
+            .
+          </Typography>
+        </Collapse>
+        <Collapse title="O que é o Código Covidografia?">
+          <Typography variant="smallBody">
+            Não necessita de um Código Covidografia para utilizar a plataforma.
+            O Código Covidografia serve uma funcionalidade ainda em fase de
+            testes, a qual permitirá a equipas médicas acompanhar os sintomas
+            dos pacientes em isolamento domiciliário de forma automática.
+          </Typography>
+        </Collapse>
+        <Collapse title="Quero ajudar no desenvolvimento da Covidografia - como devo fazer?">
+          <Typography variant="smallBody">
+            Obrigado pelo teu interesse em ajudar-nos! Procuramos developers (vê{" "}
+            <Button
+              href="https://github.com/Tech4covid19/covidografia-onboarding"
+              type="textLink"
+            >
+              aqui
+            </Button>{" "}
+            a nossa stack), marketeers e qualquer outro perfil que nos possa
+            ajudar a ajudar! Lê primeiro o nosso{" "}
+            <Button href="/perguntas-frequentes/#" type="textLink">
```

Comment:
This link is also broken

Reply:
We don't have the "Código de Conduta" paga on this PR.

Reply:
You can already put here: `codigo-conduta`

Reply:
Fixed!